vscode-extension/src/extension.ts

```diff
@@ -69,12 +70,26 @@ export async function activate(context: ExtensionContext) {
     revealOutputChannelOn: RevealOutputChannelOn.Never
   };
 
-  const disposable = new LanguageClient(
+  const client = new LanguageClient(
     'scalameta',
     'Scalameta',
     serverOptions,
     clientOptions
-  ).start();
+  );
 
-  context.subscriptions.push(disposable);
+  const restartServerCommand = commands.registerCommand("scalameta.restartServer", async () => {
+    await exec("jps | grep coursier | awk '{ print $1 }' | xargs kill");
+    const showLogsAction = "Show server logs";
+    const selectedAction = await window.showInformationMessage(
+      "Scalameta Language Server killed, it should restart in a few seconds",
+      showLogsAction
+    );
+    const myOutputChannel = window.createOutputChannel('Scalameta');
+
+    if (selectedAction === showLogsAction) {
+      client.outputChannel.show(true);
```

Comment:
The argument of `show` is `preserveFocus`. When `true` the output channel doesn't take focus.